nixos/modules/security/acme.nix

```diff
@@ -324,8 +324,8 @@ in
                           cd ${apath}
 
                           # Test that existing cert is older than new cert
-                          KEY=${spath}/certificates/${keyName}.key
-                          if [ -e $KEY -a $KEY -nt key.pem ]; then
```

Comment:
Also; please add this change's description to the commit as well.  Now the commit only talks about doc improvements but there's also a functional change

Reply:
I’ll leave it out yes, it has no consequence to use this one or another in the current 20.03 code, since there is no way to pass --reuse-key. If it’s an option, I think it would be a good idea to backport the changes from unstable, since key pinning is striclty impossible currently

Reply:
(done)